### PR TITLE
Remove GRPC Client

### PR DESCRIPTION
**Pull request by thefringeninja**
Removed: grpc client

The dotnet GRPC client has been moved, see EventStore/EventStore-Client-Dotnet#1

Partially fixes EventStore/home#115

### CODE REVIEW

File: src/EventStore.Core.Tests/Services/Transport/Grpc/AnyStreamRevisionTests.cs [hunk 1]
```diff
@@ -1,28 +1,31 @@
 using System;
 using System.Collections.Generic;
-using Xunit;
+using EventStore.Client;
```
Comment (jageall) [inline src/EventStore.Core.Tests/Services/Transport/Grpc/AnyStreamRevisionTests.cs:3]: Fix namespace

File: src/EventStore.Core/Services/Transport/Grpc/Enumerators.AllSubscriptionFiltered.cs [hunk 2]
```diff
@@ -41,7 +38,7 @@ public class AllSubscriptionFiltered : ISubscriptionEnumerator {
 			public AllSubscriptionFiltered(IPublisher bus,
 				Position? startPosition,
 				bool resolveLinks,
-				IEventFilter eventFilter,
+				EventStore.Core.Util.IEventFilter eventFilter,
```
Comment (jageall) [inline src/EventStore.Core/Services/Transport/Grpc/Enumerators.AllSubscriptionFiltered.cs:41]: Consider changing this to a better namespace location

File: src/EventStore.Core/Services/Transport/Grpc/Uuid.cs [hunk 3]
```diff
@@ -17,11 +12,11 @@ struct Uuid : IEquatable<Uuid> {
 		public static Uuid Parse(string value) => new Uuid(value);
 		public static Uuid FromInt64(long msb, long lsb) => new Uuid(msb, lsb);
 
-		internal static Uuid FromDto(Shared.UUID dto) {
+		internal static Uuid FromDto(Client.Shared.UUID dto) {
```
Comment (jageall) [inline src/EventStore.Core/Services/Transport/Grpc/Uuid.cs:15]: Is this namespace right?

Comment (thefringeninja) [reply src/EventStore.Core/Services/Transport/Grpc/Uuid.cs:15]: Doesn't matter, yet. They are not set over the wire

**Pull request merged**